rclcpp/test/rclcpp/node_interfaces/test_node_graph.cpp

```diff
@@ -137,27 +137,29 @@ TEST(TestNodeGraph, wait_for_graph_change)
   EXPECT_THROW(
     node_graph->wait_for_graph_change(event, std::chrono::milliseconds(0)),
     rclcpp::exceptions::EventNotRegisteredError);
-  rclcpp::shutdown();
 }
 
-TEST(TestNodeGraph, get_info_by_topic)
+TEST_F(TestNodeGraph, get_info_by_topic)
 {
-  rclcpp::init(0, nullptr);
   std::shared_ptr<rclcpp::Node> node = std::make_shared<rclcpp::Node>("node", "ns");
-  auto publisher = node->create_publisher<test_msgs::msg::Empty>("topic", 1);
-
+  rclcpp::QoS publisher_qos(1);
```

Comment:
@brawner nit: can be made `const` (:+1: on making it explicit!)

Reply:
Done